server/pbench/bin/index-pbench

```diff
@@ -1633,7 +1661,7 @@ def mk_run_action(ptb, options, INDEX_PREFIX, opctx):
     #if debug_time_operations: ts("mk_pbench_metadata")
     #source['pbench'] = mk_pbench_metadata(mdconf)
     if debug_time_operations: ts("mk_run_metadata")
-    source['run'] = mk_run_metadata(ptb.mdconf, ptb.md5sum, ptb.dirname, ptb.dirprefix)
+    source['run'] = mk_run_metadata(ptb.mdconf, ptb.md5sum, ptb.dirname)
```

Comment:
Was ptb.direname the same as ptb.dirprefix?

Reply:
Yes, it was.